src/include/migraphx/matcher.hpp

```diff
@@ -858,9 +858,9 @@ auto skip_broadcasts_converts(Ms... ms)
 }
 
 template <class... Ms>
-auto skip_broadcasts_transposes_contiguous(Ms... ms)
+auto skip_post_dq_ops(Ms... ms)
```

Comment:
This should probably just be moved to simplify_qdq since it will only be used there.

Reply:
Done